diff --git a/config/firewall-config.desktop.in b/config/firewall-config.desktop.in
index f8bcb81f..4d390a6b 100644
--- a/config/firewall-config.desktop.in
+++ b/config/firewall-config.desktop.in
@@ -5,7 +5,7 @@ Icon=firewall-config
 Categories=System;Settings;Security;
 # Translators: These are searchable keywords for the firewall configuration tool
 _Keywords=firewall;network;security;iptables;netfilter;
-Exec=/usr/bin/firewall-config
+Exec=/app/bin/firewall-config
 Type=Application
 StartupNotify=true
 Terminal=false

